packages/sport/src/modules/stores/betslip.tsx

```diff
@@ -130,6 +130,8 @@ const usePersistentActiveBets = ({ active, actions: { updateActive, addActive, r
   } = useUserStore();
   const { markets, transactions, blocknumber } = useDataStore();
 
+  const pendingTransactionCount = userTransactions.filter(transaction => transaction.status === TX_STATUS.PENDING);
```

Comment:
Nitpick: since this is named pendingTransactionCOUNT, it should be the .length value of the pending transactions.
Or just name this pendingTransactions, either way works.